apps/wing-console/console/app/demo/main.w

```diff
@@ -162,7 +162,6 @@ test "Add fixtures" {
   counter.inc(100);
 }
```

Comment:
Does this have to be laid out horizontally? Is it possible to distribute them throughout the space?

![Screenshot 2024-05-06 at 16 16 38](https://github.com/winglang/wing/assets/598796/82cd2f42-b944-49c8-95df-544d67611897)

Reply:
There's some configuration options we can use to hint the overall direction of the map. Can you share this app's code so I can try it?

Reply:
That's okay for now.